### PR TITLE
Adding SymfonyBundle for Loco

### PR DESCRIPTION
**Pull request by Nyholm**
Ping @theofidry. What do you think?

### CODE REVIEW

File: src/Loco/composer.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^7.0",
```
Comment (theofidry) [inline src/Loco/composer.json:12]: `7.1` 😎 (kidding)

**Review by theofidry — COMMENTED** [review]
Looks cool to me. You might want to wait to have some services to register and add integration tests first before merging it as the tricky part is always the tests, but 👍 on the idea

File: src/Loco/composer.json (hunk 1 above)
Comment (Nyholm) [reply src/Loco/composer.json:12]: =)

**Comment by Nyholm** [comment]
Agreed. Testing is tricky. I think I'll do a common integration test for all adapters.. But how and an adapter specify what symfony versions it requires?

**Comment by theofidry** [comment]
Well I would personally go for the same Symfony version for each :P Otherwise you'll need a specific `composer.json` for each which complicates a bit things :/

**Comment by Nyholm** [comment]
Hm.. Thanks.

**Pull request merged**